public/editor/scripts/editor/js/fc/project-rename.js

```diff
@@ -117,28 +117,31 @@ define(function(require) {
   }
 
   function save(context) {
-    context.saveButton.text("{{ renameProjectSavingIndicator }}");
-
-    persist.call(context, context.titleBar.val(), function(err) {
-      if(err) {
-        console.error("[Bramble] Failed to rename the project with ", err);
-        return;
-      }
+    // Do not apply save if project title remains the same
+    if (context.titleBar.val() == Project.getTitle()) {
+      editingComplete(context);
+    }
+    else {
+      context.saveButton.text("{{ renameProjectSavingIndicator }}");
 
-      Project.setTitle(context.titleBar.val(), function(err) {
-        if (err) {
-          console.error("[Bramble] Failed to update the project internally: ", err);
+      persist.call(context, context.titleBar.val(), function(err) {
+        if(err) {
+          console.error("[Bramble] Failed to rename the project with ", err);
```

Comment:
Change to ``console.error("[Bramble] Failed to rename the project. Error:  ", err);``